PayPal/src/main/java/com/braintreepayments/api/paypal/vaultedit/PayPalVaultEditResult.kt

```diff
@@ -1,22 +1,38 @@
 package com.braintreepayments.api.paypal.vaultedit
 
 import com.braintreepayments.api.ExperimentalBetaApi
+import com.braintreepayments.api.core.PostalAddress
 
 @ExperimentalBetaApi
 sealed class PayPalVaultEditResult {
 
     /**
      * The PayPal vault edit flow completed successfully.
+     *
+     * @property clientMetadataId client metadata id
```

Comment:
@jaxdesmarais do you know if there's anything specific we need to call out for the `clientMetadataId` property?

Reply:
It will be pertinent to error handling flow, so maybe we can add something like: "used to link subsequent retry attempts if payment is declined"? Essentially this value acts as a state management and will be passed into the `edit` method after the first attempt if their is a failure. What do you think, does that make it clear?

Reply:
Oh that's right, the error handling flow! I think that description is clear, let me update the PR.